ui/pages/confirmations/components/confirm/info/personal-sign/personal-sign.tsx

```diff
@@ -58,9 +58,13 @@ const PersonalSignInfo: React.FC = () => {
           <ConfirmInfoRowUrl url={currentConfirmation.msgParams.origin} />
         </ConfirmInfoAlertRow>
         {isSIWE && (
-          <ConfirmInfoRow label={t('signingInWith')}>
+          <ConfirmInfoAlertRow
+            alertKey="signingInWith"
```

Comment:
To avoid duplicate strings, there is now a `RowAlertKey` enum inside `ui/components/app/confirm/info/row/constants.ts`.

Reply:
nicee. I think it's a bit confusing RowAlertKey represents the field [key], but I think I got it https://github.com/MetaMask/metamask-extension/pull/25613

possibly we could rename these to something like 
- `RowAlertKey` → `RowAlertFieldKey` or `RowAlertField`
- `alertKey` property to `fieldKey`